apps/openmw/mwrender/camera.cpp

```diff
@@ -98,6 +98,12 @@ namespace MWRender
         osg::Vec3d position = worldMat.getTrans();
         if (!isFirstPerson())
             position.z() += mHeight * mHeightScale;
+        else {
+            if (mTrackingPtr.getRefData().getBaseNode() && !mTrackingPtr.getRefData().getBaseNode()->getParentalNodePaths().empty()){
```

Comment:
You could use local variable to store `mTrackingPtr.getRefData().getBaseNode()` so that you don't have to repeat it twice here and once below.